docs/querying/api.md

```diff
@@ -1074,6 +1074,10 @@ The following endpoint returns various cardinality statistics about the Promethe
 ```
 GET /api/v1/status/tsdb
 ```
+URL query parameters:
+- `topN=<number>`: Return top N values for each statistics. Top 10 values are returned if not specified.
```

Comment:
Since the results were already ordered anyway and this is mostly just about the number of items, how about just calling this `limit` or `max_items`?

(If we do stick with `topN` I'd go for `top_n` or something like that in snake_case, since we mostly don't use camelCase for HTTP parameters, except in one case where camel case snuck in already.)

So then (with some other wording suggestions, this would be:

```suggestion
- `limit=<number>`: Limit the number of returned items to a given number for each set of statistics. By default, 10 items are returned.
```

Reply:
I gree with limit

Reply:
Updated to 'limit'.